### DISCUSSION TITLE
Apollo Theme: Check box displays 2 boxes when checked

### DISCUSSION THREAD

**Discussion by chrishj59** [primefaces/community#1506] [PrimeReact Templates] [answered]
When the checkbox is in the checked state,  2 boxes are displayed one with primeReact styling and one unstyled.

I would expect only one checkbox displayed.
I have replicated the issue in a minimal version.

The code is on [github repo](https://github.com/chrishj59/primeReactIssue/tree/main)

I cannot see why I get the warning: 
Decide between using a controlled or uncontrolled input element and remove one of these props

The example checkbox code:
`export default function HomePage() {
	const [checked, setChecked] = useState<boolean>(false);
	return (
		<div>
			<div className="card flex justify-content-center">
				Home Page
				<Checkbox
					onChange={(e: CheckboxChangeEvent) =>
						setChecked(e.checked ? e.checked : false)
					}
					checked={checked}></Checkbox>
			</div>
		</div>
	);
}`

**Comment by melloware** [comment, marked as answer]
You need to update your theme.css. You are using old CSS

**Comment by chrishj59** [comment]
The odd thing is it only affects the CheckBox - the other components such as DataTable and PickList look as expected.

**Reply by melloware** [reply]
What theme are you using?  See: https://github.com/primefaces/primereact/issues/6222

**Reply by chrishj59** [reply]
On the actual site, I am using Apollo V10.1.0 with /theme/theme-light/purple/theme.css.  The symptoms seem similar to the 6222 that you quoted.

**Reply by melloware** [reply]
Yeah I think your Apollo theme needs updating!

**Reply by chrishj59** [reply]
I am using V10.1.0 which is the last version as per PrimeStore. Do you have any idea when the next version of the Apollo theme will be available? Thanks

**Reply by melloware** [reply]
I would ask in the `PrimeReact Templates` channel or I can move this over to PrimeReact Templates.

**Reply by chrishj59** [reply]
I am not sure how to move to PrimeReact Templates. If you could do so it would be great. Many thanks

**Reply by melloware** [reply]
Done
